Phonebook.Frontend/src/app/app.component.ts

```diff
@@ -33,11 +46,20 @@ export class AppComponent implements OnInit, OnDestroy {
     private matDialog: MatDialog,
     private platform: Platform,
     private router: Router,
-
+    public featureFlagService: FeatureFlagService,
     private activatedRoute: ActivatedRoute
-  ) {}
+  ) { }
   public ngOnInit() {
     this.store.dispatch(new InitTheme());
+    this.featureFlagService
+      .get('firstApril')
+      .pipe(untilComponentDestroyed(this))
+      .subscribe(flag => {
+        if (flag) {
+          this.store.dispatch(new SetTheme(Theme.unicorn_theme));
+        }
+      });
+    this.isUnicornThemeActive();
```

Comment:
What if somebody decides that this should be his everyday theme? He will get the message everyday...

Reply:
Maybe just move the the snack bar to the Feature flag